src/server/req_message.c

```diff
@@ -185,17 +186,16 @@ void
 req_py_spawn(struct batch_request *preq)
 {
 	int             jt;		/* job type */
+	int             js;     /* job state */
```

Comment:
Align the comment with the previous one.

Reply:
Fixed in rc 17 https://github.com/PBSPro/pbspro/pull/590/commits/f94a3405ef13edaa87df40a89af36420b9627c7b